nupic/research/frameworks/vernon/mixins/oml.py

```diff
@@ -309,7 +320,11 @@ def run_meta_testing_phase(self, num_classes_learned):
         memorize without forgetting.
         """
 
-        lr = self.find_best_lr(num_classes_learned)
+        # Decide on the lr to use.
+        if self.run_lr_sweep:
+            lr = self.find_best_lr(num_classes_learned)
+        else:
+            lr = self.lr_sweep_range[0]
```

Comment:
This defaults to 1e-1. The most common LR we've seen in the experiments, found through grid search in test time, is 1e-3. I think we should default to that.

Reply:
Good point, I'll make that change.